### DISCUSSION TITLE
Alpha and Beta

### DISCUSSION THREAD

**Discussion by Ruth-ikegah** [EddieHubCommunity/support#80] [Q&A] [answered]
Please what does it mean when someone says a feature is in beta testing?

**Comment by eddiejaoude** [comment, marked as answer]
Great question @Ruth-ikegah 👍 

In `beta` testing using means (and in this case), not everyone has access to it - our community is special, we have early access 💪  . Also it means it could be a bit buggy, any bugs/ideas/suggestions I will feed back to GitHub for them to get user feedback soon 🤓

**Reply by Ruth-ikegah** [reply]
> Great question @Ruth-ikegah 👍
> 
> In `beta` testing using means (and in this case), not everyone has access to it - our community is special, we have early access 💪 . Also it means it could be a bit buggy, any bugs/ideas/suggestions I will feed back to GitHub for them to get user feedback soon 🤓

Thanks @eddiejaoude how about alpha?

**Reply by eddiejaoude** [reply]
I think it varies from company to company (and person to person). Alpha is usually given to even less people and has more bugs 😂 

Personally I think of it as the following...

| stage | throw away | automated tests | user exposure |
| :--- | :--- | :--- | :--- |
| prototype | ✅ | ❌ | tiny |
| alpha | ❌ | ✅ | small |
| beta | ❌ | ✅ | medium |

**Comment by Ruth-ikegah** [comment]
Nice explanation! Appreciate the screenshot!

**Reply by eddiejaoude** [reply]
No problem. Great to have these discussions 🤓
